akka-http-core/src/main/scala/akka/http/scaladsl/model/ErrorInfo.scala

```diff
@@ -80,7 +80,10 @@ object EntityStreamException {
  * The limit can also be configured in code, by calling [[HttpEntity#withSizeLimit]]
  * on the entity before materializing its `dataBytes` stream.
  */
-case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
+final case class EntityStreamSizeException(limit: Long, actualSize: Option[Long] = None) extends RuntimeException {
```

Comment:
Yes and no. Technically it is not, in practice no one should have been constructing this exception in the first place other than us. 

cc @patriknw